src/tests/v1/test_moderation_events.py

```diff
@@ -32,6 +32,63 @@ def test_moderation_events_status(self):
         self.assertEqual(response.status_code, 200)
         self.assertIn('data', result)
         self.assertIsInstance(result['data'], list)
+    
+    def test_moderation_events_restriction(self):
```

Comment:
What do you think of renaming it to `test_moderation_events_access_permissions_by_user_role`?